.github/workflows/codeql-analysis.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v4
+      uses: actions/checkout@v4.1.7
```

Comment:
:thinking: because some CodeQL tags are not code-signed 4.1.7 has been pinned for now.